docs/api/api/api.txt

```diff
@@ -889,6 +889,7 @@ POST /source/<project>/<package>?cmd=commitfilelist
 Parameters:
   withvalidate: activate sha validation code (optional, but recommended)
   comment: comment for history, optional
+  keeplink: bool, optional. If the package is a link and keeplink is true, then the link is preserved. By default keeplink is false and the link is forgotten on when committing a new revision.
```

Comment:
I would not write "forgotten", but "the entire content is replaced what can replace or remove a _link file"

Reply:
Fixed